imagescripts/create_config.sh

```diff
@@ -19,19 +19,19 @@ worker_processes auto;
 error_log /dev/stdout ${LOG_LEVEL};
 
 events {
-    worker_connections 1024;
+  worker_connections 1024;
```

Comment:
It was intentional.

The files will look like this:
```
server {
  listen              7778 ssl;
  keepalive_timeout   0;
  server_name         xyz.com;

  ssl_certificate         /etc/letsencrypt/live/prism0815.myqnapcloud.com/fullchain.pem;
  ssl_certificate_key     /etc/letsencrypt/live/prism0815.myqnapcloud.com/privkey.pem;
  ssl_trusted_certificate /etc/letsencrypt/live/prism0815.myqnapcloud.com/cert.pem;
  ssl_session_timeout 1d;
  ssl_session_cache shared:SSL:50m;
```

Instead of this:
```
    server {
      listen              7778 ssl;
      keepalive_timeout   0;
      server_name         xyz.com;

      ssl_certificate         /etc/letsencrypt/live/prism0815.myqnapcloud.com/fullchain.pem;
      ssl_certificate_key     /etc/letsencrypt/live/prism0815.myqnapcloud.com/privkey.pem;
      ssl_trusted_certificate /etc/letsencrypt/live/prism0815.myqnapcloud.com/cert.pem;
      ssl_session_timeout 1d;
      ssl_session_cache shared:SSL:50m;
```

Unnecessary spaces at the beginning. Of course.. it has the same result with or without the spaces..